### PR TITLE
UA Bugfix Privacy Policy (EXPOSUREAPP-7274)

### PR DESCRIPTION
**Pull request by NataliaLemmerth**
This fixes https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7274 see 1. & 3.
1.: Missing text in English compared to German: https://github.com/corona-warn-app/cwa-app-ios/blob/release/2.3.x/src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html#L1029
3.: Minor syntax changes

PR ios: ios: https://github.com/corona-warn-app/cwa-app-ios/pull/2733

### CODE REVIEW

**Review by axelherbstreith — DISMISSED** [review]

**Review by harambasicluka — DISMISSED** [review]

File: Corona-Warn-App/src/main/assets/privacy_en.html [hunk 1]
```diff
@@ -1256,7 +1256,7 @@ <h2>12. How can you withdraw your consent?
 
 </p>
 <h3>
-    a. Consent to "exposure logging"
+    a. Consent to 'exposure logging'
```
Comment (fynngodau) [inline Corona-Warn-App/src/main/assets/privacy_en.html:1259]: Why did you change this from `""` to `''`? For the other points, it is `“”`. See https://github.com/corona-warn-app/cwa-documentation/issues/610.

**Review by InkenTownsend — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/assets/privacy_en.html [hunk 2]
```diff
@@ -1226,7 +1226,7 @@ <h2>
 </p>
 
 <p>
-    In addition, the confirmation of the authenticity of your app may involve the transfer of data
+    The EU has issued an adequacy decision for Switzerland, which determines the adequacy of the level of data protection in the country (Art. 45 GDPR). In addition, please note that users can retrieve the latest positive lists regardless of where they are (even if they are abroad on holiday or on a business trip, for example). In addition, the confirmation of the authenticity of your app may involve the transfer of data
```
Comment (fynngodau) [inline Corona-Warn-App/src/main/assets/privacy_en.html:1229]: This now sounds rather repetitive because of "In addition, […] In additon, […]". May I suggest replacing one of the two with "Furthermore, […]"?

**Review by fynngodau — CHANGES_REQUESTED** [review]
Thanks for addressing some of the current privacy policy issues.

**Review by InkenTownsend — APPROVED** [review]
UA approved

**Review by harambasicluka — APPROVED** [review]

**Review by gohumble — APPROVED** [review]
